modules/distr-tracing-tempo-install-web-console.adoc

```diff
@@ -22,18 +22,22 @@ You can install the {TempoShortName} from the *Administrator* view of the web co
 
 .. Go to *Operators* -> *OperatorHub* and search for `{TempoOperator}`.
 
-.. Select the *{TempoOperator}* that is *OpenShift Operator for Tempo* -> *Install* -> *Install* -> *View Operator*.
+.. Select the *{TempoOperator}* that is *provided by Red Hat*.
 +
 [IMPORTANT]
 ====
-This installs the Operator with the default presets:
+The following are the default presets for this Operator:
```

Comment:
```suggestion
The installation uses the following default presets for this Operator:
```

Reply:
https://www.ibm.com/docs/en/ibm-style?topic=word-usage look for "following" - do not use "following" as a noun

Reply:
Thank you. I've come up with a different suggestion:
```suggestion
The following selections are the default presets for this Operator:
```